app/components/trainees/confirmation/schools/view.html.erb

```diff
@@ -4,6 +4,11 @@
       {
         key: t(".lead_school"),
         value: lead_school_partial,
-        action: govuk_link_to(t("change"), edit_path),
+        action: govuk_link_to(t("change"), edit_lead_school_path),
+      },
+      {
```

Comment:
This section should only be present for `school_direct_salaried` trainees, `school_direct_tuition_fee` never have an employing school.